packages/synapse-react-client/src/synapse-queries/entity/useGetEntityHeaders.ts

```diff
@@ -80,15 +85,19 @@ export function useGetEntityHeaders(
 }
 
 export function useGetEntityHeader(
-  entityId: string,
+  entityId: string = '',
   versionNumber?: number,
   options?: Partial<
     UseQueryOptions<EntityHeader | undefined, SynapseClientError>
   >,
 ) {
   const { accessToken, keyFactory } = useSynapseContext()
+  if (entityId) {
+    entityId = normalizeSynPrefix(entityId)
+  }
```

Comment:
If there's a truthy (not the empty string) entityId, add `syn`.

Don't do anything with empty string, because it may be that the caller of the hook has not yet given us an ID (it could be the result of a different request). If that's the case, we should disable the query until then.